sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -62,6 +62,12 @@
 - Removed `SearchServiceCounters.SkillsetCounter`.
 - Removed `new SearchOptions(string continuationToken)` overload.
 
+## 11.0.0-preview.1 (2020-03-13)
+
+- Initial preview of the Azure.Search client library enabling you to query
+  and update documents in search indexes.
+
+[net-guidelines-collection-properties]: https://docs.microsoft.com/dotnet/standard/design-guidelines/guidelines-for-collections#collection-properties-and-return-values
```

Comment:
Please revert this.  I know the ordering looks screwy, but it's historically accurate.  preview.1 came before all the other previews, even though the package/major version changed.